pkg/divert/driver.go

```diff
@@ -15,26 +15,48 @@ package divert
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/okteto/okteto/pkg/divert/istio"
 	"github.com/okteto/okteto/pkg/divert/weaver"
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
 	"github.com/okteto/okteto/pkg/k8s/diverts"
+	"github.com/okteto/okteto/pkg/k8s/virtualservices"
 	"github.com/okteto/okteto/pkg/model"
-	appsv1 "k8s.io/api/apps/v1"
-	apiv1 "k8s.io/api/core/v1"
+	"github.com/okteto/okteto/pkg/okteto"
 	"k8s.io/client-go/kubernetes"
 )
 
 type Driver interface {
 	Deploy(ctx context.Context) error
 	Destroy(ctx context.Context) error
-	ApplyToDeployment(d1 *appsv1.Deployment, d2 *appsv1.Deployment)
-	ApplyToService(s1 *apiv1.Service, s2 *apiv1.Service)
+	ApplyNamespaceToProxy() string
```

Comment:
Probably this name doesn't correspond anymore and should be renamed to `GetDivertNamespace`. Am I wrong?